app/src/main/java/com/example/dinhh/soundscape/presentation/screens/record/RecordActivity.kt

```diff
@@ -69,10 +72,6 @@ class RecordActivity : AppCompatActivity() {
         Log.d("filename", fileName)
         myAudioRecorder!!.setOutputFile(fileName)
```

Comment:
Do we need to force unwrap myAudioRecorder!!

Reply:
Actually not anymore, since it's lateinit now.